src/interface/src/app/login/login.component.html

```diff
@@ -14,6 +14,17 @@ <h1 class="signin-title">Sign in to Planscape</h1>
           <input type="password" required formControlName="password" matInput />
         </mat-form-field>
 
+        <button
+          mat-flat-button
+          (click)="openDialog()"
+          aria-label="Opens a dialog for users who have forgotten their password"
+          i18n-aria-label="
```

Comment:
Looks like this is causing issues with some of the tests. Here's the output from one if you don't know how to access it yet:
Chrome Headless 116.0.5845.96 (Linux x86_64) LoginComponent continue as guest navigates to home page FAILED
	Error: It looks like your application or one of its dependencies is using i18n.
	Angular 9 introduced a global `$localize()` function that needs to be loaded.
	Please run `ng add @angular/localize` from the Angular CLI.
	(For non-CLI projects, add `import '@angular/localize/init';` to your `polyfills.ts` file.
	For server-side rendering applications add the import to your `main.server.ts` file.)
	    at _global.$localize (node_modules/@angular/core/fesm2020/core.mjs:29729:15)
	    at constsOrFactory (ng:///LoginComponent.js:35:18)
	    at createTView (node_modules/@angular/core/fesm2020/core.mjs:12175:60)
	    at getOrCreateComponentTView (node_modules/@angular/core/fesm2020/core.mjs:12150:28)
	    at createRootComponentView (node_modules/@angular/core/fesm2020/core.mjs:13976:49)
	    at ComponentFactory.create (node_modules/@angular/core/fesm2020/core.mjs:13835:35)
	    at apply (node_modules/@angular/core/fesm2020/testing.mjs:26224:51)
	    at _ZoneDelegate.invoke (node_modules/zone.js/fesm2015/zone.js:372:26)
	    at ProxyZoneSpec.onInvoke (node_modules/zone.js/fesm2015/zone-testing.js:287:39)
	    at _ZoneDelegate.invoke (node_modules/zone.js/fesm2015/zone.js:371:52)
	    
	    You can run these tests locally by running `ng test`

Reply:
I have fixed the tests that I broke, note that the existing tests are not in a stable state on main.